### PR TITLE
Calculating qualities of peaks

### PR DESCRIPTION
**Pull request by Daroslaw**
Simple algorithm for calculation of qualities of peaks.

**The algorithm assumes, that input is valid.**
The size of peaks container should be one element smaller than valley container.

### CODE REVIEW

File: src/Spectre.libPeakFinder.Tests/PeakQualityCalculatorTest.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/*
+* PeakQualityCalculatorTest.cpp
+* Tests for PeakQualityCalculator class.
+*
+Copyright 2018 Michal Gallus
```
Comment (Sand3r-) [inline src/Spectre.libPeakFinder.Tests/PeakQualityCalculatorTest.cpp]: Thanks for credits

**Comment by Daroslaw** [comment]
[Implement PeakQualityCalculator](https://trello.com/c/F5Cah3rF/413-implement-peakqualitycalculator)

File: src/Spectre.libPeakFinder/PeakQualityCalculator.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    /// <summary>
+    /// Calculates qualities of peaks.
+    /// </summary>
+    class PeakQualityCalculator
+    {
+    public:
+        /// <summary>
+        /// Determines qualities of peaks.
+        /// </summary>
+        /// <param name="peaksIntensities">Container of intensities of peaks.</param> // TODO Adjust this
```
Comment (Sand3r-) [inline src/Spectre.libPeakFinder/PeakQualityCalculator.h]: TODO? :)

**Review by Sand3r- — DISMISSED** [review]
Good job :>

**Review by Sand3r- — DISMISSED** [review]

**Review by Regis128 — DISMISSED** [review]

**Comment by Sand3r-** [comment]
Please update version in appveyor.yml before merging.

**Review by Sand3r- — APPROVED** [review]

**Review by Regis128 — APPROVED** [review]

**Pull request merged**